test/automation/src/positron/positronWelcome.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { Code } from '../code';
+
+const LOGO_LOCATOR = '.product-logo';
+const TITLE_LOCATOR = '.gettingStartedCategoriesContainer div.header div .positron';
+const FOOTER_LOCATOR = '.gettingStartedCategoriesContainer div.footer';
+const START_SECTION = '.positron-welcome-page-open';
+const HELP_TITLE = '.positron-welcome-page-help';
+const OPEN_SECTION = '.categories-column.categories-column-right .index-list.start-container';
+const RECENT_SECTION = '.categories-column.categories-column-right .index-list.recently-opened';
+
+const HEADING_ROLE = 'heading';
+const BUTTON_ROLE = 'button';
+const LINK_ROLE = 'link';
+
+export class PositronWelcome {
+
+	logoLocator = this.code.driver.getLocator(LOGO_LOCATOR);
+	titleLocator = this.code.driver.getLocator(TITLE_LOCATOR);
+	footerLocator = this.code.driver.getLocator(FOOTER_LOCATOR);
+	startSection = this.code.driver.getLocator(START_SECTION);
+	startTitle = this.startSection.getByRole(HEADING_ROLE);
+	startButtons = this.startSection.getByRole(BUTTON_ROLE);
+	helpSection = this.code.driver.getLocator(HELP_TITLE);
+	helpTitle = this.helpSection.getByRole(HEADING_ROLE);
+	helpLinks = this.helpSection.getByRole(LINK_ROLE);
+	openSection = this.code.driver.getLocator(OPEN_SECTION);
+	openTitle = this.openSection.getByRole(HEADING_ROLE);
+	openButtons = this.openSection.getByRole(BUTTON_ROLE);
+	recentSection = this.code.driver.getLocator(RECENT_SECTION);
+	recentTitle = this.recentSection.getByRole(HEADING_ROLE);
+	newNotebookButton = this.startButtons.getByText('New Notebook');
+	newFileButton = this.startButtons.getByText('New File');
+	newConsoleButton = this.startButtons.getByText('New Console');
+	newProjectButton = this.startButtons.getByText('New Project');
```

Comment:
nit: There's still a mixing of patterns between using a `*Locator` suffix and not. I'd be good with removing the suffix in favor of clear names. I find that the context of their use/ ide intellisense features do a good job of identifying the type. Also in theory it's more abstracted, then.